microprofile.jdt/.editorconfig

```diff
@@ -0,0 +1,6 @@
+
+[**plugin.xml]
+
+indent_style = space
```

Comment:
Why you need to change this setting?

Reply:
I wanted to format the plugin.xml. The plugin.xml are formatted with indentation as 3 spaces.  I configured this so that lemminx would use 3 spaces when formatting.